debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresTaskContext.java

```diff
@@ -100,14 +100,15 @@ protected SlotState getCurrentSlotInfo() throws SQLException {
         return slotInfo;
     }
 
-    protected ReplicationConnection createReplicationConnection() throws SQLException {
+    protected ReplicationConnection createReplicationConnection(Boolean exportSnapshot) throws SQLException {
```

Comment:
The parameter type of `exportSnapshotOnCreate()` is a primitive `boolean`, so it should be the same here for the sake of consistency and avoiding potential NPEs.